From 29706f23f7ba607771f386ae1a894482f1f52289 Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Tue, 24 Nov 2020 21:06:16 +0000
Subject: [PATCH 400/462] MANTIS 0059165 initial: Call CUSS 1.5 Shortcuts
 without extension from platform config

add module  Massai\cpp\CallCmd for
CUSS_BROWSER_CHROME.exe, CUSS_BROWSER_FIREFOX.exe, CUSS_BROWSER_IE.exe



git-svn-id: svn://localhost/SelfServiceBuild/trunk@730 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/cpp/makefile.mak | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/SelfServiceBuild/Build/cpp/makefile.mak b/SelfServiceBuild/Build/cpp/makefile.mak
index aed479d..0784f68 100644
--- a/SelfServiceBuild/Build/cpp/makefile.mak
+++ b/SelfServiceBuild/Build/cpp/makefile.mak
@@ -97,7 +97,8 @@ COMMON_DIRECTORIES= \
  Massai\idl\cuss \
  Massai\cpp\initIDL \
  Massai\cpp\textIDL \
- Massai\cpp\textIDL\static
+ Massai\cpp\textIDL\static \
+ Massai\cpp\CallCmd \
 
 PLATFORM_DIRECTORIES= \
  Massai\idl\massai \
-- 
2.41.0.windows.1

